commit af1fa9ead258f216ee97b349204d358a3d580257
Author: Michael Still <mikal@stillhq.com>
Date:   Fri Sep 25 10:39:05 2026 +1000

    Let Keystone log JSON.
    
    This is the first service to honour openstack_logging_json, so the
    demonstration is kept deliberately minimal.
    
    Change-Id: I08eb215f664ec7627e60eaf234695f9454e0daca
    Signed-off-by: Michael Still <mikal@stillhq.com>

diff --git a/ansible/roles/keystone/templates/keystone.conf.j2 b/ansible/roles/keystone/templates/keystone.conf.j2
index e13d9436d..f11a08f3a 100644
--- a/ansible/roles/keystone/templates/keystone.conf.j2
+++ b/ansible/roles/keystone/templates/keystone.conf.j2
@@ -1,5 +1,6 @@
 [DEFAULT]
 debug = {{ keystone_logging_debug }}
+use_json = {{ openstack_logging_json }}
 
 transport_url = {{ rpc_transport_url }}
 
